gunrock/graph/dynamic_graph/slabhash_graph_base.cuh

```diff
@@ -36,12 +36,14 @@ struct SlabHashGraphBase {
   using VertexT = _VertexT;
   using ValueT = _ValueT;
   using SizeT = _SizeT;
-
+  
   using HashContextT = typename std::conditional<
       REQUIRE_EDGE_VALUES,
       GpuSlabHashContext<VertexT, ValueT, SlabHashTypeT::ConcurrentMap>,
       GpuSlabHashContext<VertexT, ValueT, SlabHashTypeT::ConcurrentSet>>::type;
 
+  static constexpr uint32_t edgesPerSlab = REQUIRE_EDGE_VALUES ? 15 : 30;
```

Comment:
Consider renaming `edgesPerSlab` to `kEdgesPerSlab` to make it more obvious this is a constant throughout the code.

Reply:
Fixed https://github.com/gunrock/gunrock/pull/768/commits/b29b7db9b18319e67bae87236b8c2b60371fbde6